ferrocene/doc/internal-procedures/src/docs/signatures.rst

```diff
@@ -92,6 +92,32 @@ This will emit warnings if some documents are not signed.
    if some of the cached pages built locally are out of date. To fix them,
    remove the ``build/`` directory.
 
+Debugging signatures differences locally compared to CI
+-------------------------------------------------------
+
+The signature process relies on the documentation produced locally and the
+documentation produced in CI to be bit-by-bit identical. If there is any
+difference, the signature will not match.
+
+Debugging why signatures pass locally but fail on CI require looking at the
+differences between the two. To aid generating such a diff, we created the
+``ferrocene/tools/document-signatures/diff.py`` script. The script is supposed
+to be run by the person who signed the document.
+
+Before running the script, make sure you have a valid GitHub Personal Access
```

Comment:
Maybe change it to "valid GitHub Personal Access token (classic)" and link to https://github.com/settings/tokens .